code-challenges.js

```diff
@@ -13,22 +37,77 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+// a function that takes in array
 
+const oddSorted = (array) => {
+    // create a filter to check for type of numbers
+    let numbersOnly = array.filter(value => { 
+        return typeof value === "number"
+    })
+    // check for odds
+    let oddsOnly = numbersOnly.filter(value => {
+        return value % 2 !== 0
+    })
+    // sort 
+    let sortedOddNums = oddsOnly.sort((a,b) => a-b)
+    return sortedOddNums
```

Comment:
I love that your new variables are named really well to help to describe your intent. great job!